src/views/settings/Security/Session.vue

```diff
@@ -20,16 +20,46 @@ export default {
           [
             this.$t('Basic'),
             [
-              'SECURITY_WATERMARK_ENABLED',
               'SECURITY_SESSION_SHARE',
               'SESSION_EXPIRE_AT_BROWSER_CLOSE',
               'VIEW_ASSET_ONLINE_SESSION_INFO',
               'SECURITY_MAX_IDLE_TIME',
               'SECURITY_MAX_SESSION_TIME'
             ]
+          ],
+          [
+            this.$t('Watermark'),
+            this.$store.getters.hasValidLicense ? [
+              'SECURITY_WATERMARK_ENABLED',
+              'SECURITY_WATERMARK_SESSION_CONTENT',
+              'SECURITY_WATERMARK_CONSOLE_CONTENT',
+              'SECURITY_WATERMARK_COLOR',
+              'SECURITY_WATERMARK_FONT_SIZE',
+              'SECURITY_WATERMARK_HEIGHT',
+              'SECURITY_WATERMARK_WIDTH',
+              'SECURITY_WATERMARK_ROTATE'
+            ] : []
           ]
         ],
-        fieldsMeta: {},
+        fieldsMeta: {
+          SECURITY_WATERMARK_COLOR: {
+            component: 'el-color-picker',
+            el: {
+              size: 'small',
+              'show-alpha': true,
+              predefine: [
+                'rgba(128, 128, 128, 0.25)',
+                'rgba(0, 0, 0, 0.15)',
+                'rgba(255, 255, 255, 0.2)',
+                'rgba(176, 176, 176, 0.3)',
+                'rgba(91, 155, 213, 0.25)',
+                'rgba(192, 80, 77, 0.2)',
+                'rgba(112, 173, 71, 0.2)',
+                'rgba(255, 192, 0, 0.18)'
+              ]
+            }
+          }
+        },
         onSubmit: async(validValues) => {
           const url = '/api/v1/settings/setting/?category=security_session'
 
```

Comment:
It seems like you're discussing code-related questions related to a settings system. The first part of your input is unclear due to missing context in some areas. Can you clarify more about what parts need checking or any specific problems or needs mentioned?